homeassistant/components/notify/mysensors.py

```diff
@@ -0,0 +1,65 @@
+"""
+MySensors notification service.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/notify.mysensors/
+"""
+from homeassistant.components import mysensors
+from homeassistant.components.notify import (ATTR_TARGET,
+                                             BaseNotificationService)
+
```

Comment:
This will only be set up via discovery from the mysensors component. So I don't think the platform needs to depend explicitly on the mysensors component. If I add it will it have any consequences for the normal setup via discovery? No other mysensors platforms have this set.